CHANGELOG-developer.next.asciidoc

```diff
@@ -68,6 +68,8 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Split split httpmon out of x-pack/filebeat/input/internal/httplog. {pull}36385[36385]
 - Beats publishing pipeline does not propagate the close signal to its clients any more. It's responsibility of the user to close the pipeline client. {issue}38197[38197] {pull}38556[38556]
 - Debug log entries from the acker (`stateful ack ...` or `stateless ack ...`) removed. {pull}39672[39672]
+- Journald input now calls `journalctl` instead of using `github.com/coreos/go-systemd/v22@v22.5.0/sdjournal`, the CGO dependency has been removed from Filebeat {pull}40061[40061]
```

Comment:
Users don't care about CGO or necessarily know what it means. Write these from the perspective of someone who has never looked at the code.

Reply:
I'm a bit confused here... That's the developer's changelog as far as I remember it does not get published to our users, hence I focused on the internal changes and got very technical.

Reply:
You are correct, I am just not used to seeing the developer changelog get updated.